index.js

```diff
@@ -3,8 +3,8 @@
 var request = require('request');
 
 // log severities
-var DEBUG = "debug";
-var ERROR = "error";
+var DEBUG = 'debug';
+var ERROR = 'error';
 
 /**
  * Create an Echo client
```

Comment:
Argh github fail on my part. I misinterpreted the code fold. It looked like it was missing for addEvent. Please disregard.